site/content/docs/5.0/components/navbar.md

```diff
@@ -479,6 +480,59 @@ Also note that **`.sticky-top` uses `position: sticky`, which [isn't fully suppo
 </nav>
 {{< /example >}}
 
+## Scrolling
+
+Add `.navbar-nav-scroll` to a `.navbar-nav` to enable vertical scrolling when expanded navbar content gets too tall. By default, scrolling kicks in at `75vh` (or 75% of the viewport height), but you may override that with a local CSS custom property. For example, add `style="--height: 65vh;"` to the `.navbar-nav`.
```

Comment:
`70%`?

Reply:
Are you suggesting we change something to `70%`?

Reply:
Isn't the default 70%?